config/locales/es.yml

```
@@ -80,15 +80,15 @@ es:
       waiting_list_reminder:
         subject: "Recordatorio: estás en la lista de espera de codebar (%{date_time})"
   messages:
-    accepted_invitation: "Thanks for getting back to us %{name}. See you at the workshop!"
-    already_invited: 'You have already RSVPd or joined the waitlist for this workshop.'
+    accepted_invitation: "Gracias por comunicarte con nosotros %{name}. ¡Nos vemos en el taller!"
+    already_invited: 'Ya ha confirmado su asistencia o se ha unido a la lista de espera para este taller.'
     error_blank_note: 'Debes seleccionar una nota'
-    rejected_invitation: "We are so sad you can't make it, but thanks for letting us know %{name}."
-    already_rsvped: "You have already confirmed you attendance!"
-    not_attending_already: "You have already confirmed you can't make it!"
-    no_available_seats: "Unfortunately there are no more spaces left :(. If you would like to join our waiting list send us an email at %{email}"
-    feedback_saved: "Thank you for taking the time to give us feedback!"
-    feedback_not_found: "You have already submitted feedback for this event."
+    rejected_invitation: "Estamos tan tristes que no puedes asistir, pero gracias por hacérnoslo saber, %{name}."
+    already_rsvped: "¡Ya has confirmado tu asistencia!"
+    not_attending_already: "¡Ya has confirmado que no puedes asistir!"
+    no_available_seats: "Desafortunadamente no quedan más espacios :(. Si desea unirse a nuestra lista de espera envíenos un correo electrónico a %{email}"
+    feedback_saved: "¡Gracias por tomarse el tiempo para darnos su opinión!"
+    feedback_not_found: "Ya ha enviado comentarios para este evento".
```

There seems to be an issue with the double quote character used. I think this should resolve it.

```suggestion
    feedback_not_found: "Ya ha enviado comentarios para este evento".
```

---

@despo I think the issue is that the period is outside the double quotes, so the correct string would be:

```suggestion
    feedback_not_found: "Ya ha enviado comentarios para este evento."
```